dash-renderer/src/StoreObserver.ts

```diff
@@ -0,0 +1,113 @@
+import {
+    any,
+    filter,
+    forEach,
+    map,
+    path
+} from 'ramda';
+
+import { Store, Unsubscribe } from 'redux';
+
+type Observer<TStore> = (store: TStore) => void;
+type UnregisterObserver = () => void;
```

Comment:
Could use a third-party library to do this.. but this was simple enough to implement and sufficient for our need.. 🤷